personalwebsite/projects/templates/projects.html

```diff
@@ -7,7 +7,7 @@ <h1 class="display-5 font-family-roboto">Projects</h1>
 </div>
 {% for project in all_projects %}
```

Comment:
Before doing a for loop, it would be better to first check if `all_projects` exist.
`{% if all projects %}`
`{% for project in all_projects %}`
your content here
`{% endfor %}`
`{% endif %}`

Reply:
Okay, for this I'll do that check and then add an {% else %} block to display a message that says "No projects to display".